docs/DeveloperGuide.md

```diff
@@ -861,6 +863,60 @@ testers are expected to do more *exploratory* testing.
    4. Test case: `find` <br>
       Expected: Error details are shown.
 
+### Sorting the applications list
+
+1. Default sort setting
+
+    1. Prerequisites: Use an instance of CinternS that has never ran a `sort` command yet. Multiple applications in the list.
+
+    1. Expected: The list is being sorted in chronological order of application date.
+
+1. Sorting the application list
+
+    1. Prerequisites: Multiple different applications in the list. Multiple applications with interviews.
+
+    1. Test case: `sort`<br>
+       Expected: The applications list is sorted in chronological order of application date. This is the default sort order when no parameters are supplied to the `sort` command. The interview list should not change.
+
+    1. Test case: `sort o/company`<br>
+       Expected: The applications list is sorted in alphabetical order of company. The interview list should not change.
+
+    1. Test case: `sort r/`<br>
+       Expected: The applications list is sorted in reverse chronological order of application date. The interview list should not change.
+
+    1. Test case: `sort o/interview r/`<br>
+       Expected: The applications list is sorted in reverse chronological order of interview date. Applications with no interviews are all at the bottom of the list. The interview list should not change.
+
+    1. Test case: `sort o/email`<br>
+       Expected: The application list order does not change. Error details are shown.
+
+1. Adding applications to sorted list
+
+    1. Add an application into the list.<br>
+       Expected: It appears in the application list in the correct position such that the list remains sorted in its current order.
+
+1. Modifying interviews in sorted list
+
+    1. Prerequisites: Sort the applications list by interview (either in forward or reverse order). Multiple different applications in list. Multiple applications with interviews and without interviews.
+
+    1. Test case: Add an interview to an application without an interview.<br>
+       Expected: The application moves from near the bottom of the application list to the correct position such that the list remains sorted in order of interview date.
+
+    1. Test case: Remove an interview from an application with an interview.<br>
+       Expected: The application moves to near the bottom of the application list together with the other applications with no interview.
+
```

Comment:
The numbering is incorrect

Reply:
Oh don't worry, in markdown the list numbering doesn't affect the numbering that actually gets displayed as long as the first entry is labelled with a 1. The benefit of doing this is not having to edit later numbers when you edit one of the entries
https://www.markdownguide.org/basic-syntax/#ordered-lists